.superpowers/brainstorm/83668-1775853195/content/field-design.html

```
@@ -0,0 +1,324 @@
+  </div>
+
+  <div class="zp-field">
+    <div class="zp-field__header">
+      <span class="zp-field__name">metadata</span>
+      <span class="zp-field__type">object</span>
+      <span class="zp-field__badge zp-field__badge--optional">optional</span>
+    </div>
+    <div class="zp-field__body">Arbitrary key-value metadata attached to the user.</div>
+
+    <div class="zp-field-group--expandable is-open">
+      <button class="zp-field-group__trigger" onclick="toggleExpandable(this)">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)